extensions/amp-powr-player/validator-amp-powr-player.protoascii

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```

Comment:
Please make sure years in the licenses are the current year. In this case it should be 2018 not 2016.